content/doc/administration/requirements/java.adoc

```diff
@@ -58,3 +58,18 @@ Modern versions of Jenkins controllers and Jenkins agents verify Java requiremen
 and notify users when they are launched with an unsupported version.
 
 The plugin:versioncolumn[Versions Node Monitors plugin] provides detailed Java version monitoring.
+
+== Java Development Kits Used
+
+Jenkins uses Eclipse Temurin as its primary JDK for building and testing Java based applications.
+This includes:
+
+* Container Images
+* Controller Builds
+* Testing Infrastructure
+
+Some of the reasons for choosing Temurin are:
+
+* It's availability over a wide range of platforms, including different operating systems and architectures, over many different Java SE versions.
+* Regular maintenance and long term support provided by the Eclipse Foundation.
+* Additional performance optimizations designed to improve the performance and scalability of the JDK in enterprise-level applications.
```

Comment:
I could not find any Temurin page that claimed they were adding additional performance optimizations.  Can you point to the location where they make that claim?
```suggestion
```

Reply:
I'm sorry, I took some help from ChatGPT. I  guess I should have been more careful

Reply:
That's impressive.  The text is well phrased and the claim is believable that it might come from an organization that creates JDK distributions.  If you can find an attribution for that text, I'm happy to see it included.  Otherwise, we leave it with the first two items

Reply:
not really impressive, it's one of the biggest pitfalls of it, confidence and believable but no references and often wrong